multisig/proposals/arbitrumsepolia/vip-008/index.ts

```diff
@@ -0,0 +1,65 @@
+import { parseUnits } from "ethers/lib/utils";
+import { ethers } from "hardhat";
+
+import { NETWORK_ADDRESSES } from "../../../../src/networkAddresses";
+import { makeProposal } from "../../../../src/utils";
+
+const { arbitrumsepolia } = NETWORK_ADDRESSES;
+
+export const REWARD_DISTRIBUTOR_CORE_0 = "0x8E73FE3F7E29100Ad9d1C7F35fba2D2c823c8579";
+export const COMPTROLLER_CORE = "0xA198909e8432f262f6978F2C81B04b32c55eb063";
+export const VTREASURY = "0x4e7ab1fD841E1387Df4c91813Ae03819C33D5bdB";
+export const USDT = "0xf3118a17863996B9F2A073c9A66Faaa664355cf8";
+export const VUSDT_CORE = "0xFf3a9fD8d4f208Ed98f56A5F771AD470258866D7";
+export const ACM = "0xa36AD96441cB931D8dFEAAaC97D3FaB4B39E590F";
+
+const vip008 = () => {
+  return makeProposal([
+    {
+      target: ACM,
+      signature: "giveCallPermission(address,string,address)",
+      params: [
+        ethers.constants.AddressZero,
+        "setLastRewardingBlocks(address[],uint32[],uint32[])",
+        arbitrumsepolia.NORMAL_TIMELOCK,
+      ],
+    },
+    {
+      target: ACM,
+      signature: "giveCallPermission(address,string,address)",
+      params: [
+        ethers.constants.AddressZero,
+        "setRewardTokenSpeeds(address[],uint256[],uint256[])",
+        arbitrumsepolia.NORMAL_TIMELOCK,
+      ],
+    },
+    {
+      target: ACM,
+      signature: "giveCallPermission(address,string,address)",
+      params: [
+        ethers.constants.AddressZero,
+        "setLastRewardingBlockTimestamps(address[],uint256[],uint256[])",
+        arbitrumsepolia.NORMAL_TIMELOCK,
+      ],
+    },
+    // CORE POOL REWARDS
+    {
+      target: VTREASURY,
+      signature: "withdrawTreasuryToken(address,uint256,address)",
+      params: [USDT, parseUnits("500", 6), REWARD_DISTRIBUTOR_CORE_0],
```

Comment:
We are going to distribute 500 USDT per day ([here](https://github.com/VenusProtocol/isolated-pools/pull/396/files#diff-18185c86bd8baed36466d7a325efd11e7729ffdd43200a4808db76a6340185ff)). I would send at least 365,000 USDT to the distributor, to have enough USDT for two years

Reply:
[Done](https://github.com/VenusProtocol/vips/pull/274/commits/50a5130aa627f0bdf31135d0241bc2cc6cf7c459)